common/lib/xmodule/xmodule/css/capa/display.scss

```diff
@@ -401,10 +394,7 @@ div.problem {
 
     &.partially-correct, &.ui-icon-check {
       p.status {
-        display: inline-block;
-        width: 25px;
-        height: 20px;
-        background: url('#{$static-path}/images/partially-correct-icon.png') center center no-repeat;
+        @include status-icon($partiallycorrect, $asterisk-icon);
```

Comment:
should be consistent about snake casing, in other places this appears as `partially-correct`